package.json

```
@@ -30,6 +30,7 @@
     "remark-cli": "^12.0.1",
     "remark-frontmatter": "^5.0.0",
     "remark-gfm": "^4.0.1",
+    "remark-lint-no-trailing-spaces": "^4.0.3",
```

```suggestion
    "remark-lint-no-trailing-spaces": "=4.0.3",
```

Let's pin an exact version of this community-maintained package, just in case

---

fixed!